team_red: ["jnogueira","tmadeira", "talmeida", "jmoreira"]
team_blue: ["moliveira", "jsantos", "drato","mferreira"]
team_green: ["ttavares", "acastro", "dsilva", "psilva", "tpereira"]